spec/client/ics-008-wasm-client/README.md

```diff
@@ -262,115 +407,116 @@ The response for queries is as follows:
 ```rust
 #[cw_serde]
 pub struct QueryResponse {
-	pub status: String, // Status query response
-	#[serde(skip_serializing_if = "Option::is_none")]
-	pub genesis_metadata: Option<Vec<GenesisMetadata>>, // metadata KV pairs
+  pub status: String, // Status query response
+  #[serde(skip_serializing_if = "Option::is_none")]
+  pub genesis_metadata: Option<Vec<GenesisMetadata>>, // metadata KV pairs
 }
 
 #[cw_serde]
 pub struct GenesisMetadata {
-	pub key: Vec<u8>,
-	pub value: Vec<u8>,
+  pub key: Vec<u8>,
+  pub value: Vec<u8>,
 }
 ```
 
-#### `Contract Execute`
-Every `Wasm Contract` must support these execute messages:
+#### Contract execute
+
+Every Wasm contract must support these execute messages:
 
 ```rust
 #[cw_serde]
 pub struct VerifyMembershipMsg {
```

Comment:
Should this match the [raw or not raw message](https://github.com/ComposableFi/centauri/blob/vmarkushin/wasm/light-clients/ics10-grandpa-cw/src/msg.rs#L150-L170)?

Reply:
To me it's the "raw" version. @vmarkushin what do you think?